src/bootstrap-table.js

```diff
@@ -3150,16 +3150,32 @@
       this.updatePagination()
     }
 
-    scrollTo (_value) {
-      if (typeof _value === 'undefined') {
+    scrollTo (params) {
+      if (typeof params === 'undefined') {
         return this.$tableBody.scrollTop()
       }
 
-      let value = _value
-      if (typeof _value === 'string' && _value === 'bottom') {
-        value = this.$tableBody[0].scrollHeight
+      let options = {unit: 'px', value: 0};
+      let scrollTo = 0;
+      if (typeof params === 'object') {
+        options = $.extend(options, params);
```

Comment:
`Object.assign(options, params)`, because we plan to remove jQuery.

Reply:
Good to know, I try not to use jquery next time.